src/core/ColorPicker/ColorInputPanel.tsx

```diff
@@ -86,6 +103,7 @@ export const ColorInputPanel = React.forwardRef(
 
     const handleColorInputChange = () => {
       let color;
+      let alpha = 1;
```

Comment:
Alpha should already be part of the color, so I don't get this part.

Reply:
Removed